internal/restconfig/restconfig.go

```diff
@@ -112,6 +112,7 @@ func (rcp *Producer) WithDefaultNamespace(defaultNamespace string) *Producer {
 // with the given default value.
 // The chosen namespace will be passed to the PerContextFn used to process the context.
 func (rcp *Producer) WithPrefixedNamespace(prefix, prefixedNamespace string) *Producer {
+	rcp.prefixedDefaultNamespaces = make(map[string]*string)
```

Comment:
This should be in `NewProducer`, otherwise every call to `WithPrefixedNamespace` will clear any existing prefixed default namespace.

Reply:
or `SetupFlags`?

Reply:
That’s called after `WithPrefixedNamespace`, so it wouldn’t avoid the panic (and it would lose any prefixed default namespace as well).

Reply:
```suggestion
```